express-delete/index.js

```diff
@@ -0,0 +1,37 @@
+const express = require('express');
+const app = express();
+
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95,
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100,
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92,
+  },
+};
+
+app.get('/api/grades', (req, res) => {
+  res.json(Object.values(grades));
+});
+
+app.delete('/api/grades/:id', (req, res) => {
+  console.log(req.params.id);
+  res.sendStatus(204);
+  delete grades[req.params.id];
```

Comment:
You should delete the grade first before telling the client that it was successful 😄